bottlerocket-settings-models/settings-extensions/kubernetes/src/lib.rs

```diff
@@ -96,6 +96,8 @@ pub struct KubernetesSettingsV1 {
     node_ip: IpAddr,
     pod_infra_container_image: SingleLineString,
     hostname_override: ValidLinuxHostname,
+    ids_per_pod: KubernetesIdsPerPodValue,
```

Comment:
We use i64 directly in the shape.

https://github.com/bottlerocket-os/bottlerocket-settings-sdk/blob/93069665154c90f8ea3a01f4a78972f20dea7d8e/bottlerocket-settings-models/settings-extensions/kubernetes/src/lib.rs#L74


For consistency, let's just do:

```suggestion
    ids_per_pod: i64,
```

And we can drop the custom model type `KubernetesIdsPerPodValue`

Reply:
Synced on this - I think this is more like a preference of whether we want to let kubelet handle the validation or we do the validation in advance. 

Either way makes sense so I will not block this.